website/versioned_sidebars/version-0.67-sidebars.json

```diff
@@ -0,0 +1,79 @@
+{
+  "version-0.67-docs": {
+    "The Basics (Windows)": [
+      "version-0.67-getting-started",
+      "version-0.67-rnw-dependencies",
```

Comment:
Should there be a new versioned file with this ID in this PR?

Reply:
Yes, the getting started page should have been updated to target 0.67 with a new project, which in turn would have made a new snapshot when the new version was created by docusaurus.

Reply:
Oops l did that update but forgot to hit save. Will fix now.